AutocompleteClient/FW/Config/AutocompleteConfig.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  Constants.swift
+//  Constructor.io
+//
+//  Copyright © Constructor.io. All rights reserved.
+//  http://constructor.io/
+//
+
+import Foundation
+
+public struct AutocompleteConfig {
+
+    public let autocompleteKey: String
+    public var resultCount: AutocompleteResultCount?
+    public var clientID: String?
```

Comment:
I don't think clientID should be configurable.  That's the identifier for the client, like `cioios-<version>`.

Reply:
@nmarkovic04 ^^